docs/release_notes.rst

```diff
@@ -1,23 +1,11 @@
 .. _release-notes:
 
-=============
-Release Notes
-=============
-
-This page contains the release notes for Qiskit, starting from Qiskit 0.45, the first time that Qiskit and Qiskit Terra had the same versions.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+Qiskit |version| release notes
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 ..
-    These release notes get converted into Markdown files via the infrastructure at https://github.com/Qiskit/documentation, which then gets deployed to https://docs.quantum.ibm.com/api/qiskit/release-notes. Changes to these release notes will update those release notes the next time the API docs are generated.
-
-    To change release notes prior to Qiskit 0.45, update the Qiskit/documentation repository directly.
-
-.. release-notes::
-   :earliest-version: 0.45.0rc1
-
-.. release-notes::
-   :earliest-version: 0.45.0
-   :branch: stable/0.46
+    These release notes get converted into Markdown files via the infrastructure at https://github.com/Qiskit/documentation, which then gets deployed to https://docs.quantum.ibm.com/api/qiskit/release-notes. Changes to these release notes will update those release notes the next time the API docs are generated for this version.
 
 .. release-notes::
-   :earliest-version: 0.45.0
-   :branch: stable/0.45
+   :earliest-version: 1.0.0rc1
```

Comment:
Can you put a line in the comment above this explaining what the earliest version should be set to, so if we forget, we'll have it there?